tools/check-tx-logs

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mvn dependency:copy-dependencies
+
+java -cp "target/dependency/*:target/classes" org.neo4j.tools.txlog.CheckTxLogs $*
```

Comment:
Tool class name is `CheckTxLogs` and corresponding script file name is `check-tx-log`.
Since it's a new module would propose to define common naming strategy that should be enforced to all tools in here.
Would it make more more sense to name them without `-` like `checkTxLogs` in this case? 
@lutovich , @tinwelint  wdyt?